Navigation/Profile/PhotosTableViewCell.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  PhotosTableViewCell.swift
+//  Navigation
+//
+//  Created by Мария on 20.12.2022.
+//
+
+import Foundation
+import UIKit
+
+protocol TableCellWithCollectionDelegate: AnyObject {
+    func openImageGallery(image: UIImage)
+}
+
+class PhotosTableViewCell: UITableViewCell, UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
+    private enum Constants {
+        static let reuseIdentifier = "collection_cell"
+    }
+
+    private let titleLabel = UILabel()
+    private let viewArrow = UIImageView()
```

Comment:
Опять же не очень понятно что это за view. Будет понятнее, если будет в имени содержаться `imageView`

```suggestion
    private let arrowImageView = UIImageView()
```